README.md

```diff
@@ -1 +1,304 @@
-# budget_app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] On SignUp page, I cannot signup an account. I am getting error as shown below: 

![image](https://user-images.githubusercontent.com/58504655/204102768-85749b61-3d76-473a-892f-a361704d4250.png)

Reply:
Dear @arslanbisharat 
The Heroku error has been fixed. You can now access it. Thank you.

Reply:
Dear @arslanbisharat 
For the tech and other links. When you click them, they direct you to the same readme page baring the information.